features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -90,6 +91,12 @@
 		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.addon.ip/${project.version}</bundle>
 	</feature>
 
+	<feature name="openhab-core-config-discovery-addon-sddp" version="${project.version}">
+		<feature>openhab-core-base</feature>
+		<feature>openhab-core-config-discovery-addon</feature>
+		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.sddp/${project.version}</bundle>
```

Comment:
This needs to be:
`<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.addon.sddp/${project.version}</bundle>`

Reply:
And it probably would not hurt to keep
`<feature>openhab-core-config-discovery-sddp</feature>`
in this feature as well. It would avoid problems if the would be removed from the base feature at some point, and shouldn't hurt. I think the resolver will cope with that.

Reply:
@mherwege I made the changes that you suggest. However that causes the build to fail. I have tried building it locally with about 20 different variations. I have no idea what I am doing. And I have no idea what I am doing wrong. I give up!!